### DISCUSSION TITLE
Toast imperative api

### DISCUSSION THREAD

**Discussion by majugurci** [wix/react-native-ui-lib#1787] [Ideas]
It would be good to also have posibility to show toast component through imperative api.
It would be much easier to work with it from outside react components and even in react components. We wouldn't have to manually set visibility and render component in multiple places.

Something like this: https://github.com/calintamas/react-native-toast-message
It would only have one instance in the app and we could show message from anywhere.

**Comment by MuhammedKpln** [comment]
Well, using my own wrapper component for this purpose, made possible with imperative api. It would be better with an out-of-box solution, as it's kinda silly otherways.
